tests/lambda.egg

```diff
@@ -62,20 +62,20 @@
 (function evals-to (Term) Value)
 
 (rule ((= e (Val val)))
-      ((union (evals-to e) val)))
```

Comment:
Same here. It's an interesting choice to make evals-to a custom function instead of a constructor. This is a new pattern to me, but it seems to work.

Reply:
This is indeed a new pattern. I'll explain more during the meeting.